.github/workflows/main.yml

```diff
@@ -33,7 +33,9 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -U .[test]
-        pip install -q wagtail==${{ matrix.wagtail-version }}
+        pip install -q wagtail==${{ matrix.wagtail-version }}.*
+        pip install -q django==${{ matrix.django-version }}.*
+        pip show wagtail django # For debugging purposes
```

Comment:
The previous setup didn't seem to install the latest patches of Wagtail or Django. The pip show helps to see in the output that the correct versions are tested.